harness/ts/package.json.tmpl

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Don't love generating JSON (or other forms of structured data) from string templates but ok..

Reply:
Yeah I suppose that makes perfect sense in retrospect but this was easy to do with built-in Go stdlib stuff